modules/video/src/lkpyramid.cpp

```diff
@@ -616,9 +622,9 @@ void cv::detail::LKTrackerInvoker::operator()(const Range& range) const
                 }
             }
 
-#if CV_SSE2
+#if CV_SSE2 || CV_VSX
             float CV_DECL_ALIGNED(16) bbuf[4];
-            _mm_store_ps(bbuf, _mm_add_ps(qb0, qb1));
+            v_store_aligned(bbuf, qb0 + qb1);
             ib1 += bbuf[0] + bbuf[2];
             ib2 += bbuf[1] + bbuf[3];
```

Comment:
@terfendail Could you suggest something here to avoid temporary buffer?

Reply:
It could be done with

```
v_float32x4 qf0, qf1;
v_recombine(v_interleave_pairs(qb0 + qb1), v_setzero_f32(), qf0, qf1);
ib1 += v_reduce_sum(qf0);
ib2 += v_reduce_sum(qf1);
```